Cargo.toml

```diff
@@ -0,0 +1,11 @@
+[package]
+name = "hypothesis"
```

Comment:
Is this providing a full-fledged Hypothesis? I thought the Rust core was just the Conjecture engine, or have I missed something?

Reply:
The thing you're missing is that when using Helix the crate name is the name you require the extension under in Ruby, so you're strongly encouraged to make it the same as the path name.

But I agree it's problematic. I'll just make it something long and have two top level paths.